packages/gestalt/dist/index.d.ts

```diff
@@ -1390,21 +1425,27 @@ interface PageHeaderAction {
 interface PageHeaderProps {
   title: string;
   badge?:
-    | {
-        text: string;
-        tooltipText?: string | undefined;
-      }
-    | undefined;
+  | {
+    text: string;
+    tooltipText?: string | undefined;
+  }
+  | undefined;
   borderStyle?: 'sm' | 'none' | undefined;
   dropdownAccessibilityLabel?: string | undefined;
   helperIconButton?:
-    | {
-        accessibilityLabel: string | undefined;
-        accessibilityControls: string | undefined;
-        accessibilityExpanded: boolean | undefined;
-        onClick: ButtonEventHandlerType;
-      }
-    | undefined;
+  | {
+    accessibilityLabel: string | undefined;
+    accessibilityControls: string | undefined;
+    accessibilityExpanded: boolean | undefined;
+    onClick: AbstractEventHandler<
```

Comment:
still anchor?

Reply:
PageHeader component didn't change, so I decoupled it from Button type, that's why they're still the same type but no longer ButtonEventHandlerType (which is now only Button event)